package.json

```diff
@@ -38,7 +38,7 @@
     "@solana/wallet-adapter-sollet": "^0.8.0",
     "@solana/web3.js": "^1.30.2",
     "@tippyjs/react": "^4.2.5",
-    "@uxdprotocol/uxd-client": "^2.56.0",
+    "@uxdprotocol/uxd-client": "^3.0.0",
```

Comment:
Would not work without using the latest UXDClient

Reply:
Using the latest uxd-client, it's required to bring along the `peerDependencies` at the expected versions.

LMK if this is an issue